README.md

```diff
@@ -1132,7 +1132,7 @@ If called in an unsupported platform (Android or iOS &lt; 15), an `UnsupportedPl
 showInAppMessages(options?: { messageTypes?: IN_APP_MESSAGE_TYPE[] | undefined; } | undefined) => Promise<void>
 ```
 
-Shows in-app messages available from the App Store or Google Play. You need to disable messages from showing
+Shows in-app messages available from the App <a href="#store">Store</a> or Google Play. You need to disable messages from showing
```

Comment:
I think this Store href is not intentional. Perhaps a too wide replace? Just to confirm

Reply:
Ohh wow... this change was performed automatically by the docs library 😬 The actual code documentation here seems fine:  https://github.com/RevenueCat/purchases-capacitor/blob/55465d515f7843a73b19193c302124fb69597f60/src/definitions.ts#L869

Hmm can probably try to see what's wrong, but I don't want to block the release... I might ship with this and try to see if the same thing happens in main, and fix it there. Good catch!

Reply:
Sure! Thanks!